src/contexts/CollaborationProvider.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  createContext,
+  type PropsWithChildren,
+  useMemo,
+  useState
+} from 'react'
+import { HocuspocusProvider } from '@hocuspocus/provider'
+import { useApi, useSession } from '@/hooks'
+import { Awareness } from '@/defaults'
+
+export interface AwarenessUserData {
+  name: string
+  initials: string
+  color: string
+  avatar?: string
+  [key: string]: string | undefined
```

Comment:
It was there bcause `withYjs()` required it. But I agree. I remove it and cast in specifically for `withYjs`. Much better!

So in _views/Editor/index.tsx_ it will be `{ data: user as unknown as Record<string, unknown> }`